frontend/app/brains-management/[brainId]/components/BrainManagementTabs/components/KnowledgeTab/KnowledgeItem/UploadedKnowledgeItem.tsx

```diff
@@ -0,0 +1,28 @@
+"use client";
+
+import { UploadedKnowledge } from "@/lib/types/Knowledge";
+
+import { DeleteKnowledge } from "./DeleteKnowledge";
+import { DownloadUploadedKnowledge } from "./DownloadUploadedKnowledge";
+
+export const UploadedKnowledgeItem = ({
+  knowledge,
+}: {
```

Comment:
Seems to have many things in common with `CrawledKnowledgeItem`... 

Maybe managing `isUploadedKnowledge` in another way ?